utils/version.js

```diff
@@ -16,28 +16,59 @@ module.exports.mergeVersions = (name, versions) => {
     return versions[0];
   }
   const gitBranches = versions.filter((d) => d.includes('github') || d.includes('gitlab'));
+  const rest = versions.filter((v) => !gitBranches.includes(v));
```

Comment:
Suggestion for variable name:

```suggestion
  const gitRepos = versions.filter((d) => d.includes('github') || d.includes('gitlab'));
  const noGitRepos = versions.filter((v) => !gitBranches.includes(v));
```